runironic.sh

```diff
@@ -8,6 +8,17 @@ MARIADB_PASSWORD=${MARIADB_PASSWORD:-"change_me"}
 NUMPROC=$(cat /proc/cpuinfo  | grep "^processor" | wc -l)
 NUMWORKERS=$(( NUMPROC < 12 ? NUMPROC : 12 ))
 
+# Configuration for hardware types and interfaces
+ENABLED_HARDWARE_TYPES=${ENABLED_HARDWARE_TYPES:-"ipmi,idrac,fake-hardware"}
```

Comment:
Why not add irmc here since you're installing scciclient?

Reply:
iRMC currently does not support iPXE boot interface, if I add irmc to this configuration, then with default_boot_interface is ipxe, ironic-conductor will fail to start with error IncompatibleInterface. That's why if operator needs to use irmc, it should be passed through env variables.

Reply:
I think it has been fixed on master.

Reply:
Thanks. I've checked and seen that irmc now support ipxe. Also, installing openstack-ironic-conductor already includes python-scciclient. So I leave this patch's purpose is to allow to customize hardware types and interfaces. Adding support for irmc in ironic.conf will be done in a separated pull request.